packages/react-client/src/ReactFlightClient.js

```diff
@@ -960,6 +980,22 @@ function initializeModelChunk<T>(chunk: ResolvedModelChunk<T>): void {
         return;
       }
     }
+
+    if (__DEV__ && isReactElementOrArrayLike(value)) {
+      const debugInfo = chunk._debugInfo.splice(0);
```

Comment:
Could use a comment on this that it's giving away its debug info to the inner value.